src/Uno.UI/UI/Xaml/ElementStub.cs

```diff
@@ -198,17 +182,6 @@ private void Materialize(bool isVisibilityChanged)
 					_isMaterializing = true;
 
 					_content = SwapViews(oldView: (FrameworkElement)this, newViewProvider: ContentBuilder);
-					var targetDependencyObject = _content as DependencyObject;
```

Comment:
Changing this behavior may cause unintended consequences, as changing the visibility through a binding is supposed to be materializing the element.

Reply:
From my testing, it seems that WinUI doesn't care about the visibility at all. Setting the visibility in xaml to anything doesn't really affect whether the element is materialized or not. Binding the visibility also doesn't affect the materialization. The WinUI docs also don't mention anything about visibility. Our `SimpleDeferLoadStrategy` tests this already and on WinUi, the element that's supposed to be loaded when the visibility changes, in fact never loads.

Reply:
Got it. So it was a uno-specific behavior, which is likely not being used anymore, let's note it as a breaking change anyways, even if it's not a binary breaking change.

Reply:
Should be noted that this seems to be a breaking change only with respect to versions 5.x of Uno, because older Uno versions did also not materialize any element when changing visibility.